doc/examples/denoise_mppca.py

```diff
@@ -0,0 +1,282 @@
+"""
+======================================================
+Denoise images using the Marcenko-Pastur PCA algorithm
+======================================================
+
+The PCA-based denoising algorithm exploits the redundancy across the
+diffusion-weighted images [Manjon2013]_, [Veraart2016a]_. This algorithm has
+been shown to provide an optimal compromise between noise suppression and loss
+of anatomical information for different techniques such as DTI [Manjon2013]_,
+spherical deconvolution [Veraart2016a] and DKI [Henri2018]_.
+
+The basic idea behind the PCA-based denoising algorithm is to remove the
+data's principal components classified being mostly related to noise.
+The principal components classification can be performed based on prior noise
+variance estimates [Manjon2013]_ (see :ref:`denoise_localpca`)or automatically
+based on the Marcenko-Pastur distribution [Veraa2016a]_. In addition to noise
+suppression, the PCA algorithm can be used to estimate the noise standard
```

Comment:
In addition to the noise
suppression, the PCA algorithm can be used to get the standard deviation of the estimated noise [Veraa2016b].